src/OpenSSL/SSL.py

```diff
@@ -1501,6 +1504,29 @@ def set_cipher_list(self, cipher_list: bytes) -> None:
                 ],
             )
 
+    @_require_not_used
+    def set_ciphersuites(self, ciphersuites: bytes) -> None:
```

Comment:
@reaperhulk should this be `set_tls13_ciphersuites`, or just match the OpenSSL name?

Reply:
I changed the PR to add the tls13 in there. I hope we don't have to rename it again when TLS 1.4/2.0 comes out